File: src/main/java/br/com/alvescake/service/RevendedorService.java
package br.com.alvescake.service;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import br.com.alvescake.modelo.Revendedor;
import br.com.alvescake.repositorio.RevendedorRepositorio;

@Service
public class RevendedorService {

	@Autowired RevendedorRepositorio revendedorRepositorio;
	
	
	public Revendedor novo(Revendedor revendedor) {
        return revendedorRepositorio.novo(revendedor);	
	}
	
	public List<Revendedor> listar() {
        return revendedorRepositorio.listar();	
	}
}

File: src/main/webapp/views/revendedor/revendedor-service.js
(function () {

    angular.module('revendedor').service('RevendedorService', function ($http, BASE_URL) {
    	                                  
		var apiPostNovo='/api/revendedor/novo';
		var apiGetListar='/api/revendedor/listar';
		  
		this.novo = function (revendedor) {
	      return $http.post(BASE_URL + apiPostNovo, revendedor);
	    };
	
	    this.listar = function () {
	      return $http.get(BASE_URL + apiGetListar);
	    };

    });

})();